foo

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
LOL for this :D

Reply:
Lol, that's what you get for making temp files in whatever directory you are in and then doing a quickfix push 😆